server/coordinator.go

```diff
@@ -155,41 +159,37 @@ func (c *coordinator) drivePushOperator() {
 	}
 }
 
-func (c *coordinator) checkRegion(region *core.RegionInfo) bool {
+func (c *coordinator) checkRegion(region *core.RegionInfo) (bool, []*operator.Operator) {
 	opController := c.opController
-
+	checkerIsBusy := true
 	if op := c.learnerChecker.Check(region); op != nil {
-		if opController.AddOperator(op) {
-			return true
-		}
+		checkerIsBusy = false
+		return checkerIsBusy, []*operator.Operator{op}
```

Comment:
I think the local variable `checkerIsBusy` makes less readable. I prefer `return false, []Operator{op}` directly.

Reply:
It is acceptable for `learnerChecker`.But for other checker, there is one case that checker is not busy and region cannot through checker.